technical-reports/format/design-token.md

```diff
@@ -178,5 +178,3 @@ Tool vendors are encouraged to publicly share specifications of their extension
 <p class="ednote" title="Extensions section">
   The extensions section is not limited to vendors. All token users can add additional data in this section for their own purposes.
 </p>
-
-## More token properties TBC
```

Comment:
Opinionated: I just removed this; other properties can be added here as normal revisions to the spec